markdown_lab/network/client.py

```
@@ -0,0 +1,293 @@
+            
+        Returns:
+            Response text content
+            
+        Raises:
+            NetworkError: For all network-related failures
+        """
+        return self._request_with_retries("GET", url, **kwargs)
+
+    def head(self, url: str, **kwargs) -> requests.Response:
+        """Make HEAD request with retry logic.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)